test/compiler/views_sourceinfo.spec.js

```diff
@@ -37,19 +38,19 @@ describe('Views get info on source time bounds', function() {
         });
     }
 
-    var from = '2015-01-01T00:00:00.000Z';
-    var to = '2015-02-02T00:00:00.000Z';
-    var last = '01:00:00.000';
+    var from = new JuttleMoment({ raw: '2015-01-01T00:00:00.000Z' });
+    var to = new JuttleMoment({ raw: '2015-02-02T00:00:00.000Z' });
+    var last = new JuttleMoment.duration(1, "hour");
 
     test('read stochastic -source "cdn" | view view ', [[{from: null, to: null, last: null}]]);
-    test('read stochastic -source "cdn" -from :' + from + ': | view view ', [[{from: from, to: null, last: null}]]);
-    test('read stochastic -source "cdn" -from :' + from + ': -to :' + to + ': | view view ', [[{from: from, to: to, last: null}]]);
-    test('read stochastic -last :' + last + ': -source "cdn" | view view ', [[{from: null, to: null, last: last}]]);
+    test('read stochastic -source "cdn" -from :' + from.toJSON() + ': | view view ', [[{from: from, to: null, last: null}]]);
```

Comment:
I think `valueOf` would be slightly cleaner than `toJSON` here. This is what the compiler, `Date.toString`, and other code uses and in fact `toJSON` delegates to it.